src/test/rebaseline-p.py

```diff
@@ -109,7 +109,9 @@ def local_modifications_git(file):
   return False
 
 def local_modifications_svn(file):
-  svnstat = subprocess.Popen("svn stat %s" % file, shell=True,
+  svnstat = subprocess.Popen("svn stat %s" % file,
```

Comment:
references svn. Should this be reworked for git?

Reply:
equiv would be maybe something like `git status -uno`